libfwupdplugin/fu-device.c

```diff
@@ -3427,6 +3427,27 @@ fu_device_report_metadata_post (FuDevice *self)
 	return g_steal_pointer (&metadata);
 }
 
+/**
+ * fu_device_add_security_attrs:
+ * @self: A #FuDevice
+ * @attrs: A #FuSecurityAttrs
+ *
+ * Adds HSI security attributes.
+ *
+ * Since: 1.6.0
+ **/
+void
+fu_device_add_security_attrs (FuDevice *self, FuSecurityAttrs *attrs)
```

Comment:
so if we're having individual device security attributes instead of plugin based, that means probably re-checking all the other uses to see if there is a better way to do it too.

Reply:
I did a review of the existing plugins, and I think CPU is the only one that made sense.